sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_session_async.py

```diff
@@ -246,37 +212,22 @@ async def _outgoing_flow(self, frame=None):
         flow_frame = FlowFrame(**link_flow)
         if self.network_trace:
             _LOGGER.info("-> %r", flow_frame, extra=self.network_trace_params)
-        await self._connection._process_outgoing_frame(  # pylint: disable=protected-access
-            self.channel, flow_frame
-        )
+        await self._connection._process_outgoing_frame(self.channel, flow_frame)  # pylint: disable=protected-access
 
     async def _incoming_flow(self, frame):
         if self.network_trace:
             _LOGGER.info("<- %r", FlowFrame(*frame), extra=self.network_trace_params)
         self.next_incoming_id = frame[2]  # next_outgoing_id
-        remote_incoming_id = (
-            frame[0] or self.next_outgoing_id
-        )  #  next_incoming_id  TODO "initial-outgoing-id"
-        self.remote_incoming_window = (
-            remote_incoming_id + frame[1] - self.next_outgoing_id
-        )  # incoming_window
+        remote_incoming_id = frame[0] or self.next_outgoing_id  #  next_incoming_id  TODO "initial-outgoing-id"
+        self.remote_incoming_window = remote_incoming_id + frame[1] - self.next_outgoing_id  # incoming_window
         self.remote_outgoing_window = frame[3]  # outgoing_window
         if frame[4] is not None:  # handle
-            await self._input_handles[  # pylint: disable=protected-access
-                frame[4]
-            ]._incoming_flow(
-                frame
-            )
+            await self._input_handles[frame[4]]._incoming_flow(frame)  # pylint: disable=protected-access
         else:
             futures = []
             for link in self._output_handles.values():
-                if (
-                    self.remote_incoming_window > 0
-                    and not link._is_closed  # pylint: disable=protected-access
-                ):
-                    futures.append(
-                        link._incoming_flow(frame)  # pylint: disable=protected-access
-                    )
+                if self.remote_incoming_window > 0 and not link._is_closed:  # pylint: disable=protected-access
+                    futures.append(link._incoming_flow(frame))  # pylint: disable=protected-access
             await asyncio.gather(*futures)
```

Comment:
We're keeping the `asyncio.gather` pattern here?

Reply:
No , this needs to go as well